python_modules/dagster/dagster/_core/definitions/multi_dimensional_partitions.py

```diff
@@ -293,6 +293,12 @@ def primary_dimension(self) -> PartitionDimensionDefinition:
     def secondary_dimension(self) -> PartitionDimensionDefinition:
         return self._get_primary_and_secondary_dimension()[1]
 
+    def get_tags_from_partition_key(self, partition_key: str) -> Mapping[str, str]:
```

Comment:
Super nitpick: I'd call this "get_tags_for_partition_key", because the partition key doesn't contain the tags, we're just generating the corresponding set of tags.